tests/filesystem.go

```diff
@@ -250,18 +251,31 @@ func mountRootPartition(t *testing.T, partitions []*types.Partition) bool {
 		}
 		args := []string{partition.Device, partition.MountPath}
 		_ = mustRun(t, "mount", args...)
+
+		t.Logf("starting waiting on %s", partition.Device)
+		err := systemd.WaitOnDevices([]string{partition.Device}, "tests")
```

Comment:
You're waiting on the device your mounting after you mount it?

Reply:
Is this unnecessary? The btrfs tests pass without this check.

Reply:
This makes them fail?

Reply:
This makes them fail any runs after the first one: https://gist.github.com/dgonyeo/21de88e8eda1ae68cef8f935b294a0c5

Reply:
If you've got ideas as to why I'd be happy to investigate them